### PR TITLE
Resolved conflicts and updated readme

### PR DESCRIPTION
**Pull request by Samridhi-98**
## Related Issue 

Made require changes in the README file

Closes: #17 

### Describe the changes you've made

The top image and spacing between different sections have been changed.

## Type of change

What sort of change have you made:

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Enhancement in the existing features
- [x] Other

## Mention any unusual behaviour of your code (Write NA if not)
Any unusual behaviour of your code

## Checklist:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly whereever it was hard to understand.
- [x] My changes generate no new warnings.
- [x] I have verified/tested my code by running it locally.
**Screenshot**
sp

### CODE REVIEW

**Comment by Samridhi-98** [comment]
@Namanl2001 Please review this PR

**Comment by Namanl2001** [comment]
please add some info about https://www.github.com/Namanl2001/MERN-Gurujii-dev/tree/master/contributing.md after the note section.

**Comment by Samridhi-98** [comment]
@Namanl2001 Okay will do it.

**Comment by Samridhi-98** [comment]
@Namanl2001 Done will you please review.

**Comment by Namanl2001** [comment]
Have a look on https://github.com/Namanl2001/MERN-Gurujii-dev/blob/3484d915cc047b3a4dcc525c0c256969da7d5292/README.md various links are broken.

**Comment by Samridhi-98** [comment]
@Namanl2001  which link is broken all are working

**Comment by Namanl2001** [comment]
> 
> 
> @Namanl2001 which link is broken all are working

Screenshot taken from [here](https://github.com/Namanl2001/MERN-Gurujii-dev/blob/3d4b95552f7fd2617d7949137b0c12d1eb5ed63e/README.md)

image

**Comment by Samridhi-98** [comment]
@Namanl2001  I have recheck all the links, all are working fine here is the screenshot. It would be really great if you suggest to me how can I fixed this because it's working fine. Should I need to download each image and make a separate folder for readme images or simply remove this part. Please let me know your views on this.
broken

**Comment by Samridhi-98** [comment]
@Namanl2001 I have made a separate folder for readme images to solve the broken image link issue please look into it.

**Comment by Namanl2001** [comment]
@nikitakapoor1919 Please review this PR. Is the separate folder really needed? or only I'm not able to see the logo's without that. Also some logo's were changed by you recently, so you both can decide which logo's suits best.

**Comment by nikitakapoor1919** [comment]
@Namanl2001 okay i will do that.

**Comment by nikitakapoor1919** [comment]
@Samridhi-98 Your changes looks awesome and all logos are also cool !!!
@Namanl2001 I also can't see logos.I suggest we can host logos somewhere (on github)  and create url and upload that url in readme.

I have also created deploy button in figma and upload it on github and added its link on readme 
I have deployed one reading.svg . Check this url
eg)https://raw.githubusercontent.com/nikitakapoor1919/Buttons/fb2e224da79c2ec301e810bf33261d0e00052107/reading.svg

**Comment by Samridhi-98** [comment]
@nikitakapoor1919 I have added the image URL as you mentioned above.

File: README.md [hunk 1]
```diff
@@ -51,53 +51,55 @@ $ yarn install
...
-<br>
-<br>
-<br>
 
-### <strong>Want to test your changes on mobile phone?</strong>
+</p>
+
+<h2>Contribution <a><img src="https://github.com/Samridhi-98/MERN-Gurujii-dev/blob/master/readme_image/support(1).svg" width="3%"></a> </h2>
+<p>
+ Please check this link <a href="https://github.com/Namanl2001/MERN-Gurujii-dev/blob/master/contributing.md">contribution.md </a> before start contributing and making pull request.
```
Comment (Namanl2001) [inline README.md:75]: `Contributions are very welcome! See our CONTRIBUTING.md for more information.`

**Review by Namanl2001 — CHANGES_REQUESTED** [review]
LGTM with some small nits. I guess now we can delete the `readme_image` directory.

File: README.md (hunk 1 above)
Comment (Samridhi-98) [reply README.md:75]: Okay, will do it.

**Comment by Samridhi-98** [comment]
@Namanl2001  Done! please check.

**Comment by Namanl2001** [comment]
Thanks @nikitakapoor1919 and @Samridhi-98

**Review by Namanl2001 — APPROVED** [review]

**Pull request merged**